src/app/components/Points/DelegationPoints.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import { useDelegationsPoints } from "@/app/context/api/DelegationsPointsProvider";
+
+interface DelegationPointsProps {
+  stakingTxHash: string;
+}
+
+export const DelegationPoints: React.FC<DelegationPointsProps> = ({
+  stakingTxHash,
+}) => {
+  const { delegationPoints, isLoading } = useDelegationsPoints();
+
+  const points = delegationPoints.get(stakingTxHash);
+
+  if (isLoading) {
```

Comment:
We need to check out the UX. For example when we switch tabs, we will the "loading", but it is slow at the moment